compiler/src/dmd/README.md

```diff
@@ -47,6 +47,7 @@ Note that these groups have no strict meaning, the category assignments are a bi
 | File                                                                  | Purpose                                                              |
 |-----------------------------------------------------------------------|----------------------------------------------------------------------|
 | [lexer.d](https://github.com/dlang/dmd/blob/master/compiler/src/dmd/lexer.d)   | Convert source code into tokens for the D and ImportC parsers        |
+| [lexer.d](https://github.com/dlang/dmd/blob/master/compiler/src/dmd/location.d)| Encapsulate file/line/column info for error messages, etc.           |
```

Comment:
wrong text `[lexer.d]` should be `[location.d]`

Reply:
ok